ERROR - 2022-10-16 18:49:18 --> Error connecting to the database: Error: Class "SQLite3" not found in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\SQLite3\Connection.php:74
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\SQLite3\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\SQLite3\Connection.php(48): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\SQLite3\Connection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}
ERROR - 2022-10-16 18:49:20 --> Error connecting to the database: Error: Class "SQLite3" not found in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\SQLite3\Connection.php:74
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\SQLite3\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\SQLite3\Connection.php(48): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\SQLite3\Connection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}
ERROR - 2022-10-16 18:51:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:51:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:53:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:53:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:53:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:53:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:58:44 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 18:58:51 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:04:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:04:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:06:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:06:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:09:58 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:09:59 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:11:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\globalHeight\project\existay\project\app\Controllers\HomeController.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\HomeController->home()
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\globalHeight\project\existay\project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\globalHeight\project\existay\project\app\Controllers\HomeController.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(902): App\Controllers\HomeController->home()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\globalHeight\project\existay\project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-10-16 19:11:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `states`
ORDER BY `state` ASC', [], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\HomeController.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\HomeController->home()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-10-16 19:12:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-10-16 19:12:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '=', 'existay', 3306, '', 0)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#15 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(74): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\DatabaseTestTrait.php(54): ExampleDatabaseTest->loadDependencies()
#3 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\codeigniter4\framework\system\Test\CIUnitTestCase.php(247): ExampleDatabaseTest->setUpDatabase()
#4 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(1147): CodeIgniter\Test\CIUnitTestCase->setUp()
#5 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#6 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestCase.php(904): PHPUnit\Framework\TestResult->run(Object(ExampleDatabaseTest))
#7 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#8 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#9 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\Framework\TestSuite.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#10 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#12 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\src\TextUI\Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#13 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\phpunit\phpunit\phpunit(98): PHPUnit\TextUI\Command::main()
#14 C:\xampp\htdocs\globalHeight\project\existay\project\vendor\bin\phpunit(123): include('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-10-16 21:38:20 --> syntax error, unexpected token "."
in APPPATH\Config\Routes.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(406): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 2 SYSTEMPATH\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 SYSTEMPATH\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
 4 ROOTPATH\spark(98): CodeIgniter\CLI\Console->run()
CRITICAL - 2022-10-16 21:39:14 --> Call to undefined function Config\create_key()
in APPPATH\Config\Routes.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:01 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:02 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:02 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:02 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:03 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:03 --> Undefined constant "Config\ran"
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:08 --> Encrypter needs a starter key.
in SYSTEMPATH\Encryption\Encryption.php on line 124.
 1 SYSTEMPATH\Encryption\Encryption.php(124): CodeIgniter\Encryption\Exceptions\EncryptionException::forNeedsStarterKey()
 2 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption->initialize()
 3 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 4 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:41:53 --> CodeIgniter\Encryption\Encryption::initialize(): Argument #1 ($config) must be of type ?Config\Encryption, string given, called in C:\xampp\htdocs\globalHeight\project\existay\project\app\Config\Routes.php on line 22
in SYSTEMPATH\Encryption\Encryption.php on line 107.
 1 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption->initialize('sajid')
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:42:02 --> CodeIgniter\Encryption\Encryption::initialize(): Argument #1 ($config) must be of type ?Config\Encryption, string given, called in C:\xampp\htdocs\globalHeight\project\existay\project\app\Config\Routes.php on line 22
in SYSTEMPATH\Encryption\Encryption.php on line 107.
 1 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption->initialize('sajid', 'session')
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:42:40 --> random_bytes(): Argument #1 ($length) must be of type int, string given
in SYSTEMPATH\Encryption\Encryption.php on line 144.
 1 SYSTEMPATH\Encryption\Encryption.php(144): random_bytes('sajid')
 2 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption::createKey('sajid')
 3 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 4 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:43:40 --> Call to undefined method CodeIgniter\Encryption\Encryption::random_bytes()
in APPPATH\Config\Routes.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:43:52 --> random_bytes(): Argument #1 ($length) must be of type int, string given
in SYSTEMPATH\Encryption\Encryption.php on line 144.
 1 SYSTEMPATH\Encryption\Encryption.php(144): random_bytes('slajdsflhalk;fjdlk;aj')
 2 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption::createKey('slajdsflhalk;fjdlk;aj')
 3 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 4 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:43:56 --> random_bytes(): Argument #1 ($length) must be of type int, string given
in SYSTEMPATH\Encryption\Encryption.php on line 144.
 1 SYSTEMPATH\Encryption\Encryption.php(144): random_bytes('slajdsflhalk;fjdlk;aj')
 2 APPPATH\Config\Routes.php(22): CodeIgniter\Encryption\Encryption::createKey('slajdsflhalk;fjdlk;aj')
 3 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 4 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:51:59 --> Undefined variable $routes
in APPPATH\Config\Routes.php on line 21.
 1 APPPATH\Config\Routes.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $routes', 'APPPATH\\Config\\Routes.php', 21)
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:54:16 --> Too few arguments to function CodeIgniter\Router\Router::__construct(), 0 passed in C:\xampp\htdocs\globalHeight\project\existay\project\app\Config\Routes.php on line 20 and at least 1 expected
in SYSTEMPATH\Router\Router.php on line 121.
 1 APPPATH\Config\Routes.php(20): CodeIgniter\Router\Router->__construct()
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:54:59 --> Undefined variable $routes
in APPPATH\Config\Routes.php on line 20.
 1 APPPATH\Config\Routes.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $routes', 'APPPATH\\Config\\Routes.php', 20)
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:55:18 --> CodeIgniter\Router\Router::__construct(): Argument #1 ($routes) must be of type CodeIgniter\Router\RouteCollectionInterface, string given, called in C:\xampp\htdocs\globalHeight\project\existay\project\app\Config\Routes.php on line 20
in SYSTEMPATH\Router\Router.php on line 121.
 1 APPPATH\Config\Routes.php(20): CodeIgniter\Router\Router->__construct('apartment')
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:57:33 --> Too few arguments to function CodeIgniter\Router\Router::__construct(), 0 passed in C:\xampp\htdocs\globalHeight\project\existay\project\app\Config\Routes.php on line 19 and at least 1 expected
in SYSTEMPATH\Router\Router.php on line 121.
 1 APPPATH\Config\Routes.php(19): CodeIgniter\Router\Router->__construct()
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:59:12 --> Undefined variable $routes
in APPPATH\Config\Routes.php on line 19.
 1 APPPATH\Config\Routes.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $routes', 'APPPATH\\Config\\Routes.php', 19)
 2 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 3 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-10-16 21:59:22 --> Call to undefined method CodeIgniter\Router\RouteCollection::method()
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\CodeIgniter.php(843): CodeIgniter\CodeIgniter->Config\{closure}()
 2 SYSTEMPATH\CodeIgniter.php(444): CodeIgniter\CodeIgniter->startController()
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
